t/ui/23-audit-log.t

```diff
@@ -100,7 +100,7 @@ $search->clear;
 $search->send_keys('older:today');
 wait_for_data_table;
 @entries = $driver->find_child_elements($table, 'tbody/tr/td', 'xpath');
-is(scalar @entries, 1, 'one element when filtered by yesterday time');
+is(scalar @entries,                     1,                  'one element when filtered by yesterday time');
```

Comment:
:D This just looks ridiculous

Reply:
Yea, perltidy probably meant to align the values with the ones on the next line. 😆